Products/Zuul/infos/device.py

```diff
@@ -151,10 +158,13 @@ def _allSeverities(self):
         if not self._severities:
             # Get UUIDs for all items in the tree
             uuids = self._getUuids(self.root)
-            self._severities = dict(
-                (uuid, self._eventFacade.getSeverityName(severity).lower())
-                    for (uuid, severity) in self._eventFacade.getWorstSeverity(uuids, ignore=(SEVERITY_INFO, SEVERITY_DEBUG)).iteritems()
-            )
+            self._severities = {
+                uuid: self._eventFacade.getSeverityName(severity).lower()
+                for (uuid, severity)
+                in self._eventFacade.getWorstSeverity(
+                    uuids, ignore=(SEVERITY_INFO, SEVERITY_DEBUG)
+                ).iteritems()
```

Comment:
There's a few things going on in here that are a bit wordy.  For legibility, perhaps
```
    events = self._eventFacade.getWorstSeverity(
        uuids, ignore=(SEVERITY_INFO, SEVERITY_DEBUG)
    )
    self._severities = {
        uuids: self._eventFacade.getSeverityName(severity).lower()
        for (uuid, severity) in events.iteritems()
    }
```
as a suggestion.

Reply:
Fixed as recommended